feat: add ignorespace option to command-line (#3)
ocient/pyocient_release: README.md

# Pyocient Release Notes

## 1.0.12

> - Added `--nohistory` command-line option. Commands from the session will not be stored in the history file.

Reviewer: ```suggestion
- Added `--nohistory` command-line option. The history file does not store commands from the session.
```

> - Added `ignorespace` option to command-line history. Lines that begin with a white space character to be omitted from the history file.

Reviewer: ```suggestion
- Added `ignorespace` option to command-line history. The history file omits lines that begin with a white space character.
```
